### ISSUE TITLE
Latest Update fails to Install

### ISSUE THREAD

**Issue by Adrian281701** [FreeTubeApp/FreeTube#5471]
### Guidelines

- [X] I have encountered this bug in the [latest release of FreeTube](https://github.com/FreeTubeApp/FreeTube/releases).
- [X] I have encountered this bug in the [official downloads of FreeTube](https://github.com/FreeTubeApp/FreeTube#official-downloads).
- [X] I have searched the issue tracker for [open](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aopen+is%3Aissue) and [closed](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aissue+is%3Aclosed) issues that are similar to the bug report I want to file, without success.
- [X] I have searched the [documentation](https://docs.freetubeapp.io/) for information that matches the description of the bug I want to file, without success.
- [X] This issue contains only one bug.

### Describe the bug

During the install process for 0.21.2beta, the progress bar stops approx halway and I get an error message saying "Freetube could not be closed, please close manually and try again". Freetube is not running, either minimised or in the sys tray. Rebooted to ensure fully closed, same error. No my AVG software is reporting it as a threat and sending it off to their threat lab

### Expected Behavior

Install and update

### Issue Labels

accessibility issue

### FreeTube Version

0.21.2 beta

### Operating System Version

windows 1 pro build 19045.4717

### Installation Method

.exe

### Primary API used

Local API

### Last Known Working FreeTube Version (If Any)

0.21.1 beta

### Nightly Build

- [ ] I have encountered this bug in the latest [nightly build](https://docs.freetubeapp.io/development/nightly-builds).

**Comment by absidue** [comment]
If you selected install just for me, make sure to pick a folder that your user account has permissions to write to, that's the usual reason for that error message.

**Comment by Adrian281701** [comment]
Thanks @absidue worked a treat....weird the instal path was correct, but browsed to it anyway, and it worked. Probally a false positive but had to force AVG to let me install it, never had that before. Anyways it says it has sent the file to AVG threat labs again. But it is working so thanks

**Comment by absidue** [comment]
It's possible that because it was only just released, AVG doesn't trust it yet.

**Comment by absidue** [comment]
Glad it's working again. I'll close this issue then as you've found a solution.

**Comment by Adrian281701** [comment]
@absidue Ref AVG, could be, and es pease close as its appears to be working. Thanks for your help
